### DISCUSSION TITLE
Where is full normal documentation?

### DISCUSSION THREAD

**Discussion by rvstron** [sukona/Grapevine#236] [Q&A]
Sorry, i cant find the grapevine documentation v.4.2.2.

**Comment by scottoffen** [comment]
This is what there is (much of which is just stubs): https://scottoffen.github.io/grapevine-legacy/en/
You can take a look at the sample project, too: https://github.com/scottoffen/grapevine-legacy/blob/master/src/Grapevine.Local/Program.cs
